doc/samtools-ampliconclip.1

```diff
@@ -0,0 +1,116 @@
+'\" t
+.TH samtools-ampliconclip 1 "6 December 2019" "samtools-1.10" "Bioinformatics tools"
+.SH NAME
+samtools ampliconclip \- clip reads using a BED file
+.\"
+.\" Copyright (C) 2008-2011, 2013-2020 Genome Research Ltd.
+.\" Portions copyright (C) 2010, 2011 Broad Institute.
+.\"
+.\" Author: Heng Li <lh3@sanger.ac.uk>
+.\" Author: Joshua C. Randall <jcrandall@alum.mit.edu>
```

Comment:
You should give yourself some credit here, @whitwham

Reply:
Ah yes, I just copied over the boilerplate from another file.

Reply:
You used variable numbers > 1 character so it's definitenly not Heng's :-)